src/main/java/br/com/caelum/clines/api/aircraftmodels/AircraftModelService.java

```diff
@@ -0,0 +1,19 @@
+package br.com.caelum.clines.api.aircraftmodels;
+
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+public class AircraftModelService {
+    private final AircraftModelRepository repository;
+    private final AircraftModelFormMapper formMapper;
+
+    public Long createAircraftModelBy(AircraftModelForm form) {
+        var aircraftModel = formMapper.map(form);
```

Comment:
It makes sense register two models with same description? Is there any way to validate this case?